src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -119,7 +119,8 @@
         DELAYED_FOR_WIFI,
         LOCAL_FILE_NOT_FOUND,
         SERVICE_UNAVAILABLE,
-        SPECIFIC_SERVICE_UNAVAILABLE
+        SPECIFIC_SERVICE_UNAVAILABLE,
+        SPECIFIC_UNSUPPORTED_MEDIA_TYPE
```

Comment:
I've used SPECIFIC_... instead of UNSUPPORTED_MEDIA_TYPE just in case we want to handle a common 415 error in the future. Besides, I think that SPECIFIC_UNSUPPORTED_MEDIA_TYPE is more appropriated because we are setting it when the response body contains a **specific** error message.